lib/internal/http2/core.js

```diff
@@ -1764,6 +1764,8 @@ class Http2Stream extends Duplex {
       throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'code', 'number');
     if (code < 0 || code > kMaxInt)
       throw new errors.RangeError('ERR_OUT_OF_RANGE', 'code');
+    if (callback && typeof callback !== 'function')
```

Comment:
Can you change the first part of this line to `callback !== undefined`

Reply:
Done!